sklearn/linear_model/tests/test_randomized_l1.py

```diff
@@ -75,6 +76,23 @@ def test_randomized_lasso():
     assert_raises(ValueError, clf.fit, X, y)
 
 
+def test_randomized_lasso_precompute():
+    # Check randomized lasso for different values of precompute
+    n_resampling = 20
+    alpha = 1
+    random_state = 42
+
+    clf = RandomizedLasso(alpha=alpha, random_state=random_state,
+                          precompute=True, n_resampling=n_resampling)
+    feature_scores_1 = clf.fit(X, y).scores_
+
+    for precompute in [False, None, 'auto']:
+        clf = RandomizedLasso(alpha=alpha, random_state=random_state,
+                              precompute=precompute, n_resampling=n_resampling)
+        feature_scores_2 = clf.fit(X, y).scores_
+        assert_array_equal(feature_scores_1, feature_scores_2)
+
```

Comment:
In Randomized lasso, `precompute = array_like` is used only for the computation of `alpha` when `alpha='aic'` or `'bic'`.
But I can definitely test it.